Updated TrustedHosts commands

SteveBurkettNZ opened the pull request:
Correct a non-standard quote mark that was in the ASRDeploymentPlanner.exe 'Store the list of VMs in a file' example command line.

azurecla commented:
Hi __@qwergram__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@qwergram : Thanks for your contribution to the Azure documentation! The author, @thraka, has been notified to review your proposed change.

Thraka commented:
@KatieCumming This is good.

Thraka commented:
@Ja-Dunn

Thraka commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

SteveBurkettNZ committed:
Corrected dodgy quote mark in commandline

Correct a non-standard quote mark that was in the ASRDeploymentPlanner.exe 'Store the list of VMs in a file' example command line.
